configuration/logs.mdx

```diff
@@ -33,19 +36,22 @@ in your project is add it to your `Cargo.toml`.
 
 ## Setting Log Level
 
-The global subscriber has an `env_filter` which defaults to the `INFO` log level if no `RUST_LOG` variable is set. You can change the log level for local runs
-with `RUST_LOG="<crate-name>=trace" cargo shuttle run`. We do not currently allow changing the log level for production deployments, and it will be set to `DEBUG` by default.
+The global subscriber has an `env_filter` which defaults to the `INFO` log level if no `RUST_LOG` variable is set. You can change the log level for local runs with `RUST_LOG="<crate-name>=trace" cargo shuttle run`.
+
+We do not currently allow changing the log level for deployments. Deployments use `RUST_LOG=debug,shuttle=trace,h2=warn` at the moment.
 
 ## Viewing Logs
 
-We have a few commands that are useful for viewing your application's logs. 
+We have a few commands that are useful for viewing your application's logs.
 
-To view the logs for your current deployment, in your project directory run:
+To view the logs for your current `running` deployment, if there is one:
 
 ```bash
 cargo shuttle logs
 ```
 
+*Caveat: The above command finds the oldest deployment in the `running` state (there should only be one...).*
```

Comment:
I may be overthinking this, but I feel this part makes it seem like we don't take this bug seriously, which could be frustrating for users. If we keep this caveat here I think we should rather link to an issue tracking it, or a faq entry, where they can find how to work around it as well.
```suggestion
*Caveat: The above command finds the oldest deployment in the `running` state*
```